src/utils/setLocalStorage.js

```diff
@@ -0,0 +1,19 @@
+export const setLocalStorage = (currentQuestion, quizData, key) => {
+  let answeredQuestions = localStorage.getItem('answeredQuestions');
+  currentQuestion.selected = key;
+  if (answeredQuestions) {
+    answeredQuestions = JSON.parse(answeredQuestions);
+    answeredQuestions[quizData.currentQuestionIndex] = currentQuestion.selected;
```

Comment:
If you only use and need `quizData.currentQuestionIndex` and `currentQuestion.selected`  so why don't you just send it within the arguments as follows:

```
export const setLocalStorage = (selectedQuestion, questionIndex, key) => {..}

setLocalStorage(currentQuestion.selected, quizData.currentQuestionIndex, key)
```